### PR TITLE
ci: speed up github actions

### PR DESCRIPTION
**Pull request by k-nasa**

### CODE REVIEW

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -117,15 +129,3 @@ jobs:
 
     - name: Docs
       run: cargo doc --features docs
-
```
Comment (dignifiedquire) [inline .github/workflows/ci.yml:120]: why remove clippy?

Comment (k-nasa) [reply .github/workflows/ci.yml:120]: I don't think there is any point in leaving this comment out. If you need a clippy later, you can just add it again.

Comment (dignifiedquire) [reply .github/workflows/ci.yml:120]: right, makes sense

**Comment by dignifiedquire** [comment]
Thanks

**Pull request merged**